aws/logs_monitoring/enhanced_lambda_metrics.py

```diff
@@ -41,6 +41,9 @@
     + r"Max\s+Memory\s+Used:\s+(?P<{}>\d+)\s+MB".format(MAX_MEMORY_USED_METRIC_NAME)
 )
 
+#Pull memorysize tag and cold start from report
+TAGS_TO_PARSE_FROM_REPORT = [MEMORY_ALLOCATED_FIELD_NAME]
```

Comment:
It looks like this doesn't include cold start.

Reply:
I was planning on doing both at once but decided to just do this one first to ensure it worked properly.

Reply:
Alright cool. It might make sense to make a note of that in the comment on line 44, just in case you don't get back to it. Otherwise 👍